src/bcf_clean_overpar.cpp

```diff
@@ -246,24 +246,6 @@ List bcfoverparRcppClean(NumericVector y_, NumericVector z_, NumericVector w_,
   di_mod.x = &x_mod[0]; 
   di_mod.y = r_mod; //the y for each draw will be the residual
 
-
-  //  }
-  //--------------------------------------------------
-  //storage for ouput
-  //in sample fit
-  /*
-  //out of sample fit
-  double* ppredmean=0; //posterior mean for prediction
-  double* fpredtemp=0; //temporary fit vector to compute prediction
-  if(dip.n) {
-  ppredmean = new double[dip.n];
-  fpredtemp = new double[dip.n];
-  for(size_t i=0;i<dip.n;i++) ppredmean[i]=0.0;
-  }
-  //for sigma draw
-  double rss, restemp;
-  */
-
```

Comment:
Is it a problem to lose this?

Reply:
Nah this is just a comment, I'm not sure what it was doing